source/what-is-the-bi-connector.txt

```diff
@@ -98,21 +99,20 @@ the BI Connector's :doc:`mongosqld process </reference/mongosqld>`.
    :figwidth: 700px
    :align: center
 
-On Premises Database and |bi-short|
------------------------------------
+On-Premises Database and |bi-short|
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Your organization may have installations of both MongoDB and |bi-short|
```

Comment:
Use [might](https://docs.mongodb.com/meta/style-guide/terminology/alphabetical-terms/#term-might) instead of [may](https://docs.mongodb.com/meta/style-guide/terminology/alphabetical-terms/#term-may) as recommended in the style guide.